wiki/Introduction-to-Creating-a-Job.md

```diff
@@ -0,0 +1,1818 @@
+it easier to narrow in on the root cause of an error. This process has faster iterations than submitting the Job
+to your distributed compute cluster, can be done in a cafe or airplane without an internet connection, and does not even require
+that you have a compute cluster that supports Open Job Description.
+
+### 2.1. Starting with shell scripts
+
+An Open Job Description Job is, essentially, describing a set of command-line commands to run, the inputs to those commands,
+and the order in which they need to be run. We encourage you to think of authoring a Job as writing a series of shell
+scripts that accomplish the objective of the Job when they are run in the correct order with the correct inputs.
+
+In this guide, you are creating a Job that generates frames of an animation using Blender and then encodes the frames into an animation
+using FFmpeg. You'll break this Job down into two steps:
+
+1. Generate the animation frames with Blender; and then
+2. Encode the generated frames into an mp4 video with FFmpeg.
+
+Note that this walkthrough is written for a bash-compatible shell such as those available on typical Linux or
+MacOS workstations. If you are using Windows, then we trust that the modifications to powershell or batch from bash
+will be straightforward enough for you to handle; though you may also run bash scripts on Windows if you install
+an application such as [Git BASH](https://gitforwindows.org/).
+
+#### 2.1.1. Creating a shell script to render with Blender
+
+First, you'll create a shell script that runs Blender to render a part of the animation.
+Looking up the [documentation for the Blender command line](https://docs.blender.org/manual/en/latest/advanced/command_line/arguments.html)
+we created the following as a starting point for our Blender script:
+
+```bash
+#!/bin/bash
+
+# Return an error code if any command in the script fails.
+set -eou pipefail
+
+# Use Blender's scripting interface to reduce the scene resolution and sampling rate to speed up testing.
+# Remove this after testing is complete.
+TMPFILE=$(mktemp)
+cat > "$TMPFILE" << EOF
+import bpy
+for s in bpy.data.scenes:
+  s.render.resolution_x = 480
+  s.render.resolution_y = 270
+bpy.context.scene.cycles.samples = 100
+EOF
+
+# Run blender to generate the image frames.
+blender --background 3d/pavillon_barcelone_v1.2.blend \
+  --python "$TMPFILE" \
+  --render-output output_frames/frame-### \
+  --render-format PNG --use-extension 1 \
+  --render-frame "1..2"
+
+# Cleanup the temporary python script
+rm -f $TMPFILE
+```
+
+This script makes some assumptions, such as being run from a directory that contains our input and output, that we'll modify next, but
+this is a good start. It uses Blender to render two frames of the 380 frame animation created by
+the ["Barcelona Pavillion" demo scene](https://www.blender.org/download/demo-files/#cycles) (accessed July 2024) at reduced resolution.
```

Comment:
```suggestion
the ["Barcelona Pavillion" demo scene](https://www.blender.org/download/demo-files/#cycles) by eMirage, distributed under CC-BY (accessed July 2024).
```

Reduced res has already been explained above. While we don't HAVE to attribute here (since we're not displaying the results of the file or using it directly, we should play it safe.